### PR TITLE
Runtime toggle

### PR DESCRIPTION
**Pull request by engine-workflow**
## Summary
Introduce helper layer which will determine at launch time whether or not to include the Java agent in JAVA_OPTS. Java only!

## Use Cases
As a user, I want to disable Datadog integration in lower environments but still promote a single image through a deployment pipeline.

### CODE REVIEW

**Comment by dmikusa** [comment]
I think this looks good but I would like to run a few tests before merging. I also need to check if we need to hold off on this until we release #162.

File: datadog/java_agent.go
Comment (dmikusa) [inline datadog/java_agent.go:43]: I think this is the culprit to the failing test. You're removing this env variable, but the test is still looking for it.

**Comment by dmikusa** [comment]
Did some tests and it all looks good. Just need to fix that unit test and we're good to merge.

File: helper/toggle.go
Comment (pivotal-david-osullivan) [inline helper/toggle.go:46]: You might be able to simply this func with [sherpa.ResolveBool](https://github.com/paketo-buildpacks/libpak/blob/main/sherpa/env_var.go#L57)?
It should allow for various truthy values or return false for unset. Since you wouldn't have to handle an error either you could reduce the func to one line

File: datadog/java_agent.go
Comment (engine-workflow) [reply datadog/java_agent.go:43]: ugh, should have caught that :/ fixing...

File: helper/toggle.go
Comment (engine-workflow) [reply helper/toggle.go:46]: nice, agreed--will change while I'm fixing the test...

File: datadog/build.go
Comment (dmikusa) [inline datadog/build.go:39]:
```suggestion
	cr, err := libpak.NewConfigurationResolver(context.Buildpack, &b.Logger)
```

**Review by dmikusa — APPROVED** [review]

**Pull request merged**
